deployments.json

```diff
@@ -1043,6 +1043,11 @@
     "name": "re.al",
     "chainId": 111188,
     "url": "https://explorer.re.al/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract"
+  }, 
+  {
+    "name": "Merlin Mainnet",
+    "chainId": 4200,
+    "url": "https://scan.merlinchain.io/address/0xa8dcebad1ea2fcf86de386462c14d7629237cc88"
```

Comment:
Is Merlin EVM compatible? I typically only merge non-standard addresses when a chain cannot support deployment to the standard address of 0xcA11bde05977b3631167028862bE2a173976CA11